lib/ast-converter.js

```diff
@@ -646,9 +663,17 @@ module.exports = function(ast, extra) {
             // Declarations
 
             case SyntaxKind.FunctionDeclaration:
+                // TODO: Discuss - this approach doesn't seem right...
+                // In the test this Identifier should not have trailingComments so naively delete
+                // - Just because it doesn't in this test, doesn't mean it couldn't have them?
+                var functionId = convertChild(node.name);
```

Comment:
I'm general, nodes don't have `trailingComments` or `leadingComments` if they are empty. If that's a pain, we could try leaving them for now.